app/core/router.py

```diff
@@ -0,0 +1,5 @@
+from users.views import UserList
+from rest_framework import routers
+
+router = routers.DefaultRouter()
+router.register('users', UserList)
```

Comment:
This logic should be implemented in the existing urls file, no need to create separate file for this